onnxruntime/core/providers/cuda/cuda_call.cc

```diff
@@ -30,6 +35,19 @@ const char* CudaErrString<cudaError_t>(cudaError_t x) {
   return cudaGetErrorString(x);
 }
 
+template <>
+const char* CudaErrString<cudnn_frontend::error_t>(cudnn_frontend::error_t x) {
```

Comment:
if you really have to return a const char* instead of std::string, please return a std::unique_ptr